src/components/Table/TableHead/TableHead.jsx

```diff
@@ -147,9 +150,43 @@ const TableHead = ({
   closeMenuText,
   lightweight,
   i18n,
+  hasResize,
 }) => {
   const filterBarActive = activeBar === 'filter';
-
+  const utilsVar ={
+    startX: 0,
+    pressed: false,
+    index : 0
+  }
+  const [state, setState] = useState({TableHeader:[]});
+  const elementsRef = useRef(ordering.map(() => createRef()));
+  useEffect(() => {
+    const nextWidth = elementsRef.current.map(
```

Comment:
These variable names need to be clearer here. The table is a huge component so we need to be explicit in our naming to really convey what it is we are trying to do and what values these variables hold.

Reply:
Ok